lib/rubocop/cop/style/redundant_parentheses.rb

```diff
@@ -27,8 +27,8 @@ class RedundantParentheses < Base
 
         def on_begin(node)
           return if !parentheses?(node) || parens_allowed?(node)
-          return if node.parent && (node.parent.while_post_type? ||
-                                    node.parent.until_post_type?)
+          return if node.parent && (node.parent.while_post_type? || node.parent.until_post_type? ||
```

Comment:
Probably that big condition can become a helper method as well.

Reply:
I updated this PR, but I'm not sure due to there may be a better approach (or name) 💦